src/sentry/templates/sentry/groups/public_details.html

```diff
@@ -1,65 +0,0 @@
-{% extends "sentry/groups/details.html" %}
```

Comment:
This file is still referenced, I didn't check to see if those references are valid